[2025-10-17T08:35:04.685+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-17T08:35:04.700+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:34:00+00:00 [queued]>
[2025-10-17T08:35:04.705+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:34:00+00:00 [queued]>
[2025-10-17T08:35:04.705+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-17T08:35:04.715+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_pwd> on 2025-10-17 08:34:00+00:00
[2025-10-17T08:35:04.721+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=728) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-17T08:35:04.723+0000] {standard_task_runner.py:63} INFO - Started process 731 to run task
[2025-10-17T08:35:04.723+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag3', 'task_pwd', 'scheduled__2025-10-17T08:34:00+00:00', '--job-id', '182', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmpjm7rj4n5']
[2025-10-17T08:35:04.725+0000] {standard_task_runner.py:91} INFO - Job 182: Subtask task_pwd
[2025-10-17T08:35:04.737+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-10-17T08:35:04.776+0000] {task_command.py:426} INFO - Running <TaskInstance: dag3.task_pwd scheduled__2025-10-17T08:34:00+00:00 [running]> on host 9565199fb8e8
[2025-10-17T08:35:04.850+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag3' AIRFLOW_CTX_TASK_ID='task_pwd' AIRFLOW_CTX_EXECUTION_DATE='2025-10-17T08:34:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-17T08:34:00+00:00'
[2025-10-17T08:35:04.851+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-17T08:35:04.852+0000] {logging_mixin.py:188} INFO - 1
[2025-10-17T08:35:04.853+0000] {logging_mixin.py:188} INFO - 2
[2025-10-17T08:35:04.853+0000] {logging_mixin.py:188} INFO - 3
[2025-10-17T08:35:04.854+0000] {logging_mixin.py:188} INFO - 4
[2025-10-17T08:35:04.854+0000] {logging_mixin.py:188} INFO - 5
[2025-10-17T08:35:04.854+0000] {logging_mixin.py:188} INFO - 6
[2025-10-17T08:35:04.855+0000] {logging_mixin.py:188} INFO - 7
[2025-10-17T08:35:04.856+0000] {logging_mixin.py:188} INFO - 8
[2025-10-17T08:35:04.857+0000] {logging_mixin.py:188} INFO - 9
[2025-10-17T08:35:04.858+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-17T08:35:04.859+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-17T08:35:04.867+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag3, task_id=task_pwd, run_id=scheduled__2025-10-17T08:34:00+00:00, execution_date=20251017T083400, start_date=20251017T083504, end_date=20251017T083504
[2025-10-17T08:35:05.138+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-17T08:35:05.150+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-17T08:35:05.152+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
